source/scripts/script.js

```diff
@@ -19,54 +16,63 @@ async function init() {
         }
     });
 
+    // Add click event listener for search button
+    const searchButton = document.getElementById('search-button');
+    searchButton.addEventListener('click', async () => {
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/cse110-fa21-group23/cse110-fa21-group23/pullRequest?prid=8408391)
```suggestion
    searchButton.addEventListener("click", async () => {
```